examples/benchmark/benchmark_detectors.py

```diff
@@ -38,12 +42,17 @@
 db_slug = cfg['db_slug']
 timestamp = time.strftime('%Y_%m_%d__%H_%M_%S')
 csv_filepath = outfile_dir / f'{benchmark}__{db_slug}__{timestamp}.csv'
-print(f'Storing results to {csv_filepath.resolve()}')
+print(f'Storing results to {csv_filepath.resolve()}.')
 
 data_dir = Path(cfg.get('data_dir', '~/.sleepecg/datasets')).expanduser()
 records = list(reader_dispatch(db_slug, data_dir))
 print(f'Loaded {len(records)} records from {db_slug}.')
 
+if cfg.get('export_records', False):
+    from sleepecg import export_record
+    print(f'Exporting records to {outfile_dir.resolve()}.')
+    for record in records:
+        export_record(record, outfile_dir / f'{record.id}-{record.lead}.txt')
```

Comment:
```suggestion
        export_record(record, outfile_dir / f'{record.id}-{record.lead}')
```

`export_record` appends `.csv`, so the file would be called e.g. `100-II.txt.csv`

Reply:
We could also be a bit more strict, i.e. throw an error if a file has an extension other than `.csv`. We could append `.csv` only if there is no extension.

Reply:
I like that!